src/layouts/no105.rs

```diff
@@ -0,0 +1,223 @@
+                    DecodedKey::Unicode('3')
+                }
+            }
+            KeyCode::Key4 => {
+                if modifiers.is_shifted() {
+                    DecodedKey::Unicode('¤')
+                } else if modifiers.is_altgr() {
+                    DecodedKey::Unicode('$')
+                } else {
+                    DecodedKey::Unicode('4')
+                }
+            }
+            KeyCode::Key5 => {
+                if modifiers.is_shifted() {
+                    DecodedKey::Unicode('%')
+                } else if modifiers.is_altgr() {
+                    DecodedKey::Unicode('€')
+                } else {
+                    DecodedKey::Unicode('5')
+                }
+            }
+            KeyCode::Key6 => {
+                if modifiers.is_shifted() {
+                    DecodedKey::Unicode('&')
+                } else {
+                    DecodedKey::Unicode('6')
+                }
+            }
+            KeyCode::Key7 => {
+                if modifiers.is_shifted() {
+                    DecodedKey::Unicode('/')
+                } else if modifiers.is_altgr() {
+                    DecodedKey::Unicode('{')
+                } else {
+                    DecodedKey::Unicode('7')
+                }
+            }
+            KeyCode::Key8 => {
+                if modifiers.is_shifted() {
+                    DecodedKey::Unicode('(')
+                } else if modifiers.is_altgr() {
+                    DecodedKey::Unicode('[')
+                } else {
+                    DecodedKey::Unicode('8')
+                }
+            }
+            KeyCode::Key9 => {
+                if modifiers.is_shifted() {
+                    DecodedKey::Unicode(')')
+                } else if modifiers.is_altgr() {
+                    DecodedKey::Unicode(']')
+                } else {
+                    DecodedKey::Unicode('9')
+                }
+            }
+            KeyCode::Key0 => {
+                if modifiers.is_shifted() {
+                    DecodedKey::Unicode('=')
+                } else if modifiers.is_altgr() {
+                    DecodedKey::Unicode('}')
+                } else {
+                    DecodedKey::Unicode('0')
+                }
+            }
+            KeyCode::OemMinus => {
+                if modifiers.is_shifted() {
+                    DecodedKey::Unicode('?')
+                } else {
+                    DecodedKey::Unicode('+')
+                }
+            }
+            KeyCode::OemPlus => {
+                if modifiers.is_shifted() {
+                    DecodedKey::Unicode('`')
+                } else if modifiers.is_altgr() {
+                    DecodedKey::Unicode('´')
+                } else {
+                    DecodedKey::Unicode('\\')
+                }
+            }
+            KeyCode::Backspace => DecodedKey::Unicode(0x08.into()),
+            KeyCode::Tab => DecodedKey::Unicode(0x09.into()),
+            KeyCode::Q => {
```

Comment:
Isn't this the same as a US 'Q'?

Reply:
Removed the `KeyCode::Q` handling in 2624803.